mongodb.install

```diff
@@ -9,63 +9,121 @@
  * Implements hook_requirements().
  */
 function mongodb_requirements($phase) {
-  $extension_name = 'mongo';
-
+  global $extension_name;
```

Comment:
Please avoid using globals: they are almost never needed, and a generic name like this one could conflict with any other contrib or custom module using the same generic global name. The reason why this was a local variable instead of a global variable or define-d constant is precisely to avoid such conflicts ; and the reason why it wasn't a class constant is because the autoloader is not reliably usable during hook_requirements if `$phase != "runtime"`.